### PR TITLE
5811 integration tests

### PR DESCRIPTION
**Pull request by mrinsin**

### CODE REVIEW

File: web-client/integration-tests/docketClerkServesPaperFiledDocumentFromCaseDetailDocumentView.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    jest.setTimeout(40000);
+    global.window.pdfjsObj = {
+      getData: () => Promise.resolve(new Uint8Array(fakeFile)),
+    };
+  });
+
+  loginAs(test, 'petitioner@example.com');
+  petitionerChoosesProcedureType(test, { procedureType: 'Regular' });
+  petitionerChoosesCaseType(test);
+  petitionerCreatesNewCase(test, fakeFile);
```
Comment (rachaelparris) [inline web-client/integration-tests/docketClerkServesPaperFiledDocumentFromCaseDetailDocumentView.test.js]: Nit: can use `uploadPetition` in `helpers.js` here. This is not wrong, though!

File: web-client/integration-tests/journey/docketClerkViewsCaseDetailDocumentView.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { runCompute } from 'cerebral/test';
+import { withAppContextDecorator } from '../../src/withAppContext';
+
+const formattedCaseDetail = withAppContextDecorator(
+  formattedCaseDetailComputed,
+);
+
+export const docketClerkViewsCaseDetailDocumentView = (
+  test,
+  docketNumber = null,
```
Comment (rachaelparris) [inline web-client/integration-tests/journey/docketClerkViewsCaseDetailDocumentView.js]: Nit: I don't think this is necessary, I don't see it used anywhere.

File: web-client/integration-tests/docketClerkServesPaperFiledDocumentFromCaseDetailDocumentView.test.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  },
+});
+test.draftOrders = [];
+
+describe('Docket Clerk Serves Paper Filed Document From Case Detail Documents View', () => {
+  beforeAll(() => {
+    jest.setTimeout(40000);
+    global.window.pdfjsObj = {
+      getData: () => Promise.resolve(new Uint8Array(fakeFile)),
+    };
```
Comment (kkoskelin) [inline web-client/integration-tests/docketClerkServesPaperFiledDocumentFromCaseDetailDocumentView.test.js:18]: I wonder if one could add `pdfjsObj` to the globals in `jest-integration.config.js` -- won't block this PR, though.

**Review by codyseibert — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

File: web-client/integration-tests/docketClerkServesPaperFiledDocumentFromCaseDetailDocumentView.test.js (hunk 3 above)
Comment (mrinsin) [reply web-client/integration-tests/docketClerkServesPaperFiledDocumentFromCaseDetailDocumentView.test.js:18]: I like the idea though, will try it out when doing DOD for the next story!

**Pull request merged**